app/serializers/export/contact_serializer.rb

```diff
@@ -1,12 +1,28 @@
 module Export
   class ContactSerializer < ActiveModel::Serializer
     attributes :sharing,
-               :receiving,
+               :following,
```

Comment:
I just realized, that you renamed that, probably because it is also named `following` in the federation gem. But these are different things:

* `sharing`: I share with that person and the person can view my private profile, send me private messages, ...
* `following`: I have this persons posts in my stream (currently always true for diaspora, the plan with adding `sharing` and `following` to the `contact` entity in the protocol was, that you can only follow the posts, without sharing your information, or you maybe want to share with the person, but this person posts much, so you don't want that in your stream)
* `receiving`: The other person shares with me (other direction of `sharing`)

So the diaspora DB currently only has `sharing` and `receiving`.

I think we should rename `following` back to `receiving` here. And we can add a new `following` that's hardcoded `true` here (because for data currently exported it is `true`).

We can also add a `subscribed` that is always `true`, for the other direction of `following` (so when the other person subscribed to us). We can then replace the hardcoded `true` with the correct value when the diaspora DB supports `following` and `subscribed`.

And we should definitely document that somewhere, what `sharing`, `following`, `receiving` and `subscribed` means.